src/Workspaces/Core/Portable/Execution/AbstractReferenceSerializationService.cs

```diff
@@ -92,6 +94,14 @@ public void WriteTo(AnalyzerReference reference, ObjectWriter writer, Cancellati
             var file = reference as AnalyzerFileReference;
             if (file != null)
             {
+                // fail to load analyzer assembly
+                var assemblyPath = TryGetAnalyzerAssemblyPath(file);
```

Comment:
Ah ok, so we do actually need the path to loaded analyzer assembly - makes sense.